modules/rosa-troubleshooting-commands.adoc

```diff
@@ -0,0 +1,80 @@
+:_module-type: REFERENCE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-troubleshoot-cli.adoc
+
+[id="rosa-troubleshooting_{context}"]
+= Troubleshooting with rosa
```

Comment:
You might want to distinguish "with the rosa CLI" for clarity.

Reply:
Done.